src/app/services/dto-transform.service.ts

```diff
@@ -0,0 +1,81 @@
+import { Injectable } from '@angular/core';
+import { MovieDTO } from '../types/DTO/movie-dto';
+import { PersonDTO } from '../types/DTO/person-dto';
+import { TvShowDTO } from '../types/DTO/tv-show-dto';
+import { MediaType } from '../types/media-type';
+import { Movie } from '../types/movie';
+import { Person } from '../types/person';
+import { TvShow } from '../types/tv-show';
+import { ApiGenresService } from './api-genres.service';
+import { Genre } from '../types/genres';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class DtoTransformService {
+  constructor(private apiGenresService: ApiGenresService) {}
+  transformPerson(dtoPerson: PersonDTO): Person {
+    const { id, profile_path, name } = dtoPerson;
```

Comment:
nit: move object destructuring to function

```
transformPerson({id, profile_path, name}: PersonDTO): Person {
```

Reply:
will do.

Reply:
Fixed